src/components/screen/ChangePw/index.tsx

```diff
@@ -13,7 +13,7 @@ import React, {
   useState,
 } from 'react';
 
-import { Button } from '@dooboo-ui/native';
+import Button from '../../shared/Button';
```

Comment:
Just a quick look. Please do not add this since we want to migrate all our shared components into `@dooboo-ui/native`. This is in our roadmap.

Reply:
I understand, the `shared/Button` actually uses `@dooboo-ui/native`'s Button.
`shared/Button` is just a wrapper providing customized shadow for `hackatalk`

Reply:
Also I think providing a wrapper component (as a decorator) in between library and real app is a good pattern if we would like to give app-wide customized features! let me know how you think @hyochan

Reply:
Ohh wait, are you saying not to use `shared` folder and move it to somewhere else?

Reply:
Sounds interesting. I want to hear further roadmap of your thought. Could you tell bit more about this?